Register employer last

Elfaram opened the pull request.

Elfaram committed:
saving to change branch

Elfaram committed:
changed page in component

Elfaram committed:
add page register, component registeremployerlast created, front ok

Commit:
Merge branch 'dev' into RegisterEmployerLast

Commit:
Merge branch 'dev' into RegisterEmployerLast

LuigiMenez commented on frontend/src/components/RegisterEmployerLast/index.jsx:
```diff
@@ -0,0 +1,61 @@
+  const [categoryJur, setCategoryJur] = useState("");
+  const [nin, setNin] = useState("");
+
+  const hChangeSiret = (evt) => {
```
Possibility to make one function instead of 3

LoicBrassart replied:
When @LuigiMenez  (or yours truly) asks for a refactoring, please do it before re-submitting your PR

SDecambray replied:
it's done

LuigiMenez reviewed (changes requested):
Blank line & optimisation function

LuigiMenez reviewed (dismissed):
Ok for me.

LoicBrassart commented on frontend/src/App.jsx:
```diff
@@ -1,16 +1,20 @@
+import "./App.css";
 import Home from "@pages/Home";
 
+
```
Useless empty line -> Prettier didn't pass through this code?

Elfaram replied:
used anytimes "npm run fix"  before commit and push...don't understand why...

LoicBrassart commented on frontend/src/components/RegisterEmployerLast/index.jsx:
```diff
@@ -0,0 +1,61 @@
+            <input
+              className="inputForm"
+              type="text"
+              name="categorie juridique"
```
Not sure it's a great idea to have spaces in a `name` attribute (or any attribute, really)

LoicBrassart commented on frontend/src/components/RegisterEmployerLast/index.jsx:
```diff
@@ -0,0 +1,61 @@
+            <input
+              className="inputForm"
+              type="text"
+              name="raison sociale"
```
Same (+ plz name everything in English)

SDecambray replied:
name it's in english now, placeholder still in french because it's a french website

LoicBrassart reviewed (changes requested):
Please fix all of these

Elfaram committed:
Update index.jsx

changed

Elfaram committed:
Update index.jsx

changed

Elfaram reviewed (commented):
fixed

Commit:
Update App.jsx

delete blank space

LuigiMenez commented on frontend/src/App.jsx:
```diff
@@ -1,18 +1,21 @@
+import "./App.css";
 import Home from "@pages/Home";
 
+
```
Blank Line

SDecambray replied:
done

LuigiMenez reviewed (changes requested):
Useless Line

Commit:
Merge branch 'dev' into RegisterEmployerLast

Commit:
Update App.jsx

Delete blank line

SDecambray reviewed (approved):
Delete blank line

Commit:
Update index.jsx

Change name in english, placeholder still in french because it's a french website

SDecambray reviewed (dismissed):
Resolve conflict with dev app.jsx and delete blank space

LuigiMenez commented on frontend/src/components/RegisterEmployerLast/index.jsx:
```diff
@@ -0,0 +1,59 @@
+              type="text"
+              name="socialReason"
+              placeholder="Raison Sociale"
+              value={formData.nin}
```
more logical : `value={formData.socialReason}` or `name='nin' `

LuigiMenez reviewed (changes requested):
Fix minor

Elfaram committed:
Update index.jsx

Elfaram reviewed (commented):
Changed Nin

LuigiMenez reviewed (approved):
Ok for me.

SDecambray reviewed (dismissed):
Ok for me

LuigiMenez reviewed (changes requested):
Fix Conflict

Commit:
Merge branch 'dev' into RegisterEmployerLast

ghost commented:
conflict fixed

LuigiMenez commented on frontend/src/App.jsx:
```diff
@@ -12,9 +13,9 @@ function App() {
       <Route path="/profile" element={<Profile />} />
       <Route path="/profile/:profileId" element={<Profile />} />
+      <Route path="/register" element={<Register />} /> 
+      <Navbar />
```
why navbar is in the router and a closing div is missing

SDecambray replied:
Tired, sorry for that

LuigiMenez reviewed (changes requested):
why navbar is in the router and a closing div is missing

Commit:
Update App.jsx

Fix with the closing dev tag and put components outside routes

LuigiMenez commented on frontend/src/App.jsx:
```diff
@@ -12,9 +13,11 @@ function App() {
     </Routes>
     <Navbar />
-  </div>
+    </div>
```
Indentation please

LuigiMenez reviewed (changes requested):
Fix indentation
